dom.bs

```diff
@@ -639,10 +639,18 @@ dictionary EventInit {
 <p>An {{Event}} object is simply named an <dfn export id=concept-event>event</dfn>. It allows for
 signaling that something has occurred, e.g., that an image has completed downloading.</p>
 
+<p>An <a>event</a> has an associated <dfn export for=Event>relatedTarget</dfn> (an
+{{EventTarget}} object). Unless stated otherwise it is null.</p>
+
+<p>An <a>event</a> has an associated <dfn export>set the relatedTarget</dfn> algorithm,
+which takes an {{EventTarget}} <var>relatedTarget</var>. Unless specified otherwise it does
+nothing.</p>
```

Comment:
So this algorithm exists because UI events need it and they need to do something different between normal UI events and Touch events?

Also, how does this work for synthetic events? I guess the event class defines it?

It's probably worth having a note here that points to the various implementations of this algorithm.